Tools/SelectionTool/SelectionInput.cs

```diff
@@ -9,5 +9,6 @@ public class SelectionInput : ActionMapInput {
 		
 		public ButtonInputControl @multiSelect { get { return (ButtonInputControl)this[0]; } }
 		public ButtonInputControl @select { get { return (ButtonInputControl)this[1]; } }
+		public ButtonInputControl @multiSelectAlt { get { return (ButtonInputControl)this[2]; } }
```

Comment:
No. At least not as far as I know.

We have one VRDevice which covers Vive and Oculus. In fact, it includes a Button2 which Vive doesn't have. Extending this to include more VRDevices was a TODO for when input stuff coalesced, but I think we also still want to default to a generic VR device so that you don't have to specify support for individual hardware.